src/UmbracoIdentity/ExternalLoginStore.cs

```diff
@@ -29,28 +30,22 @@ internal class ExternalLoginStore : DisposableObject, IExternalLoginStore
         public static readonly object Locker = new object();
         private readonly UmbracoDatabase _db;
 
-        private const string ConnString = @"Data Source=|DataDirectory|\UmbracoIdentity.sdf;Flush Interval=1;";
+        private readonly ConnectionStringSettings identityStoreConnection = ConfigurationManager.ConnectionStrings["UmbracoIdentityStore"];
```

Comment:
The main plan will be to host these tables in the same DB as Umbraco. It should use the same Database instance and then it will be cross db compatible since Umbraco takes care of all of this for us.